frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -1574,6 +1600,32 @@ With this feature enabled, the current page is factored in, resulting in the cou
             {
                 text = _("Item max width"),
                 sub_item_table = {
+                    {
+                        text_func = function()
+                            return T(_("Book-author item: %1".. "\u{202F}%"), self.settings.book_author_max_width_pct)
```

Comment:
Sorry, can you remove the concatenation, here and in two other items.
`"Book-author item: %1\u{202F}%"`